docs/embedding/sdk/questions.md

```diff
@@ -82,6 +82,7 @@ export default function App() {
 | onBeforeSave          | `() => void`                                                         | (optional) A callback function that triggers before saving. Only relevant when `isSaveEnabled = true`.                                                                                                                                                                                                             |
 | onSave                | `() => void`                                                         | (optional) A callback function that triggers when a user saves the question. Only relevant when `isSaveEnabled = true`.                                                                                                                                                                                            |
 | saveToCollectionId    | number                                                               | (optional) The target collection to save the question to. This will hide the collection picker from the save modal. Only applicable to static questions.                                                                                                                                                           |
+| initialSqlParameters  | `Record<string, string \| string[]>`                                 | (optional) A mapping of Learn more about [SQL parameters names](../../questions/native-editor/sql-parameters.md) to parameter values to set on the question, such as `{ product_id: "42" }`.                                                                                                                       |
```

Comment:
Note that `initialSqlParameters` is also exposed for static questions in https://github.com/metabase/metabase/pull/50656 -- I'm merging that PR later today.